CHANGELOG.asciidoc

```diff
@@ -64,6 +64,7 @@ TinkerPop 3.2.5 (Release Date: NOT OFFICIALLY RELEASED YET)
 * Improved error handling of compilation failures for very large or highly parameterized script sent to Gremlin Server.
 * Fixed a bug in `RangeByIsCountStrategy` that changed the meaning of inner traversals.
 * Improved Gremlin-Python Driver implementation by adding a threaded client with basic connection pooling and support for pluggable websocket clients.
+* StarVertexProperty will check meta-properties are valid.
```

Comment:
This should be written as:

Fixed a bug in `StarVertexProperty` where properties are now validated.